py/mpconfig.h

```diff
@@ -1814,6 +1814,11 @@ typedef double mp_float_t;
 #define MICROPY_PY_SSL_FINALISER (MICROPY_ENABLE_FINALISER)
 #endif
 
+// Whether to add a root pointer for the current ssl object
+#ifndef MICROPY_PY_SSL_MBEDTLS_NEED_ACTIVE_CONTEXT
+#define MICROPY_PY_SSL_MBEDTLS_NEED_ACTIVE_CONTEXT (MICROPY_PY_SSL_ECDSA_SIGN_ALT)
+#endif
+
```

Comment:
Can't this just be a REGISTER_ROOT_POINTER in modtls_mbedtls.c ?

Reply:
No, there needs to be a separate variable per thread.  Because each thread may be using a different `SSLContext` with a different callback.

Reply:
Ah I see.